src/commcare_cloud/terraform/modules/backup/main.tf

```diff
@@ -106,28 +104,12 @@ resource "aws_backup_vault_policy" "business_continuity_remote_vault_policy" {
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Effect": "Allow",
-            "Principal": "*",
-            "Action": "backup:CopyIntoBackupVault",
-            "Resource": "*",
-            "Condition": {
-                "StringEquals": {
-                    "aws:PrincipalOrgID": "${var.org_id}"
-                }
-            }
-        },
-        {
-            "Sid": "0cfc19f4-bec0-4a37-8bf8-6d15b3650f66",
+            "Sid": "Allow ${var.outside_account_id} to copy into ${aws_backup_vault.business_continuity_remote_vault.name}",
             "Effect": "Allow",
             "Principal": {
-                "AWS": "AROATDKQYKLTTBKSMZN3Z"
+                "AWS": "arn:aws:iam::${var.outside_account_id}:root"
             },
-            "Action": [
-                "backup:CopyIntoBackupVault",
-                "backup:CopyFromBackupVault",
-                "backup:DeleteRecoveryPoint",
-                "backup:ListRecoveryPointsByBackupVault"
-            ],
+            "Action": "backup:CopyIntoBackupVault",
             "Resource": "*"
         }
```

Comment:
Should we drop this entirely? I don't think the outside account needs to be able to copy anything into this vault.
The original policy was made I think to allow it to copy from this vault to its own and then delete what was copied, and the `CopyIntoBackupVault` permission was just a requirement for the others.

Reply:
I think the idea here is to allow us to copy a snapshot from the outside account into the main account, as part of a recovery step for example. I agree the way things are currently set up it's not likely to be useful, but I think it also doesn't hurt. Happy to delete it, but I'm also trying to keep these changes minimal to start with.